.buildkite/create_integration_testing_worksheet.py

```diff
@@ -101,6 +101,7 @@ def create_artifact(str_arg):
     file = open(txt_path, 'w')
     file.write(str_arg)
     file.close()
+    os.chmod(txt_path, 0o777)
```

Comment:
I'm also wondering about the error from `make docker-clean` that says `Failed to remove "/var/lib/buildkite-agent/builds/vader-1/learningequality/kolibri" (remove /var/lib/buildkite-agent/builds/vader-1/learningequality/kolibri/dist/spreadsheet-link.txt: permission denied)`.

Since normally we didn't have issues removing files generated inside the Docker containers, I think that you should remove this part?